server/src/main/java/org/opensearch/rest/BaseRestHandler.java

```diff
@@ -203,27 +202,19 @@ protected Set<String> responseParams() {
         return Collections.emptySet();
     }
 
+    protected static final String DUPLICATE_PARAMETER_ERROR_MESSAGE =
+        "Please only use one of the request parameters [master_timeout, cluster_manager_timeout].";
+
```

Comment:
Note: I moved the constant String outside the method to be directly used in RestIndicesAction and RestAllocationAction classes in below. In the 2 classes, the method      public static void parseDeprecatedMasterTimeoutParameter() can fit the requirement.